presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -0,0 +1,231 @@
+           └─ LocalExchange[HASH][$hashvalue_4] (""d_date_sk"")
+              │   Layout: [d_date_sk:bigint, $hashvalue_4:bigint]
+              │   Estimates: {rows: 0 (0B), cpu: 0, memory: 0B, network: 0B}
+              └─ RemoteSource[2]
+                     Layout: [d_date_sk:bigint, $hashvalue_5:bigint]
+
+    Fragment 2 [SOURCE]
+        Output layout: [d_date_sk, $hashvalue_6]
+        Output partitioning: BROADCAST []
+        Stage Execution Strategy: UNGROUPED_EXECUTION
+        ScanFilterProject[table = hive:default:date_dim, grouped = false, filterPredicate = ((""d_following_holiday"" = CAST('Y' AS char(1))) AND (""d_year"" = 2000))]
+            Layout: [d_date_sk:bigint, $hashvalue_6:bigint]
+            Estimates: {rows: 0 (0B), cpu: 0, memory: 0B, network: 0B}/{rows: 0 (0B), cpu: 0, memory: 0B, network: 0B}/{rows: 0 (0B), cpu: 0, memory: 0B, network: 0B}
+            $hashvalue_6 := combine_hash(bigint '0', COALESCE(""$operator$hash_code""(""d_date_sk""), 0))
+            d_following_holiday := d_following_holiday:char(1):REGULAR
+            d_date_sk := d_date_sk:bigint:REGULAR
+            d_year := d_year:int:REGULAR
+
+
+During execution of a query with dynamic filters, Presto populates statistics
+about dynamic filters in the QueryInfo JSON available through the web-ui.
+In the ``queryStats`` section, dynamic filtering stats can be found in
+``dynamicFiltersStats`` structure.
+
+.. code-block:: none
+
+    "dynamicFiltersStats" : {
+          "dynamicFilterDomainStats" : [ {
+            "dynamicFilterId" : "df_370",
+            "simplifiedDomain" : "[ [[2451546, 2451905]] ]",
+            "rangeCount" : 3,
+            "discreteValuesCount" : 0
+          } ],
+          "lazyDynamicFilters" : 1,
+          "replicatedDynamicFilters" : 1,
+          "totalDynamicFilters" : 1,
+          "dynamicFiltersCompleted" : 1
+    }
+
+Push down of dynamic filters into a table scan can be
+verified by looking at operator statistics for that table scan.
+``dynamicFilterSplitsProcessed`` records the number of splits
+processed after a dynamic filter was pushed down to the table scan.
+
+.. code-block:: none
+
+    "operatorType" : "ScanFilterAndProjectOperator",
+    "totalDrivers" : 1,
+    "addInputCalls" : 762,
+    "addInputWall" : "0.00ns",
+    "addInputCpu" : "0.00ns",
+    "physicalInputDataSize" : "0B",
+    "physicalInputPositions" : 28800991,
+    "inputPositions" : 28800991,
+    "dynamicFilterSplitsProcessed" : 1,
+
+Delaying execution for dynamic filters
+---------------------------------------
+
+It can often be beneficial to wait for collection of dynamic filters
+from build side to complete before executing the probe side table scan.
+This extra wait time can potentially result in significant overall savings
+in query runtime if dynamic filtering is able to eliminate a large amount
+of unnecessary data from the probe side.
+For the Hive connector, probe side table scan can be delayed for a configured
```

Comment:
remove `probe side` just keep `For the Hive connector, table scan can be delayed...`